.github/workflows/release-build.yaml

```diff
@@ -0,0 +1,47 @@
+name: Docker Build Publish
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - main
+
+env:
+  GITHUB_PAT: "${{ secrets.KIVA_ROBOT_GITHUB_PAT }}"
+  AWS_REGION: "us-west-2"
+  AWS_ACCESS_KEY_ID: "${{ secrets.AWS_ACCESS_KEY_ID }}"
+  AWS_SECRET_ACCESS_KEY: "${{ secrets.AWS_SECRET_ACCESS_KEY }}"
+  SLACK_BOT_TOKEN: "${{ secrets.SLACK_TOKEN }}"
+  SLACK_CHANNEL: "eng-build-failures"
+
+jobs:
+  # Build and push the image to ECR
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Checkout actions
+        uses: actions/checkout@v3
+        with:
+          repository: kiva/github-actions
+          token: ${{ env.GITHUB_PAT }}
+          path: .github/
+      - uses: snow-actions/sparse-checkout@v1.2.0
+        with:
+          repository: kiva/marketplace-web-ui-ci
+          token: ${{ env.GITHUB_PAT }}
+          path: .docker
+          patterns: |
+            resources/org/kiva/marketplaceWebUiCi/ui
+      - name: move files
+        run: |
+          mv .docker/resources/org/kiva/marketplaceWebUiCi/ui/Dockerfile .
```

Comment:
I don't see why not! If this is part of the new release flow for the project, I think moving away from relying on `marketplace-web-ui-ci` makes sense. @mcstover any concerns with that change?

Reply:
It should be, yes. As long as you pass the final ("releasable") stage to the GitHub Actions workflow so it knows, it should work.

Reply:
I don't think we should move the Dockerfile here if we can just copy it from marketplaceWebUiCi, because I would prefer that we keep our deployment pipeline and configs in a private repo instead of this public repo.  We used to have the Dockerfile here, but we moved it to marketplaceWebUiCi a while ago for that very reason.

Reply:
Also, I'm curious to know why we think we should move away from using the marketplaceWebUiCi repo?

Reply:
That's a good point about keeping some of this stuff out of this public repo.

That said, I think the move to standardizing some of our tasks in our gh actions will reduce the need for the other config repo but may not remove it entirely...

Reply:
If we move back to including `marketplaceWebUiCi` we may need to stand up another Dockerfile to refine it into this new process so we don't disrupt current prod deployments

Reply:
@emuvente I think part of the motivation is to standardize our release flow so that the monolith, services, and frontend applications follow the same/similar deployment processes. If we'd prefer to keep the Dockerfile out of this repo, I can go back to copying it over from `marketplaceWebUiCi` 👍🏻

Reply:
If we need to build a common base image, maybe we can add it here? https://github.com/kiva/base-docker-images

Reply:
> If we need to build a common base image, maybe we can add it here? https://github.com/kiva/base-docker-images

It would be great to do that so that we code keep a consistent node version across the frontend apps.